scripts/ft-benchmark-data-sender.py

```diff
@@ -111,6 +111,11 @@ def commit_to_db(data: dict) -> None:
                         type=int,
                         required=True,
                         help='Number of users')
+    parser.add_argument('--user', type=str, default='unknown', help='User name')
+    parser.add_argument('--context',
+                        type=str,
+                        default='unknown',
```

Comment:
nit: Perhaps make `user` and `context` required arguments without default, to make sure these fields have meaningful values.

Reply:
Maybe I agree, but maybe some time we will prefer to see row in db with `unknown, unknown` rather than see nothing. If you strongly confident that we want this arguments be required let me know I will change them